### PR TITLE
chore: fix keys

### PR DESCRIPTION
**Pull request by MrPink**
assign dev and prod GCS keys

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -40,6 +40,13 @@ if [[ ${TRAVIS_TAG} =~ ^[0-9]+(\.[0-9]+){2}$ ]]; then
     GCS_PROJECT="eha-data"
     GCR_PROJECT="production-228613"
 
+    openssl aes-256-cbc \
+    -K $encrypted_17d8de6bf835_key \
```
Comment (obdulia-losantos) [inline scripts/deploy.sh]: indent :grimacing:

File: scripts/deploy.sh [hunk 2]
```diff
@@ -55,14 +62,22 @@ elif [[ ${TRAVIS_BRANCH} =~ ^release\-[0-9]+\.[0-9]+$ ]]; then
...
+    RELEASE_BUCKET="aether-releases"
 
 else
 
     DOCKER_VERSION="alpha"
     GCR_VERSION=${TRAVIS_COMMIT}
     GCS_PROJECT="alpha"
     GCR_PROJECT="development-223016"
-
+    RELEASE_BUCKET="aether-releases"
```
Comment (obdulia-losantos) [inline scripts/deploy.sh]: it's the same in all cases :thinking:

File: scripts/deploy.sh [hunk 3]
```diff
@@ -55,14 +62,22 @@ elif [[ ${TRAVIS_BRANCH} =~ ^release\-[0-9]+\.[0-9]+$ ]]; then
...
 
 else
 
     DOCKER_VERSION="alpha"
     GCR_VERSION=${TRAVIS_COMMIT}
     GCS_PROJECT="alpha"
     GCR_PROJECT="development-223016"
-
+    RELEASE_BUCKET="aether-releases-dev"
```
Comment (obdulia-losantos) [inline scripts/deploy.sh]: I'm not sure if you need the `export` option. I think I tried without it and the `push-app-version` didn't get it :thinking:

**Review by obdulia-losantos — DISMISSED** [review]

File: scripts/deploy.sh [hunk 4]
```diff
@@ -39,6 +38,14 @@ if [[ ${TRAVIS_TAG} =~ ^[0-9]+(\.[0-9]+){2}$ ]]; then
     GCR_VERSION=${TRAVIS_TAG}
     GCS_PROJECT="eha-data"
     GCR_PROJECT="production-228613"
+    RELEASE_BUCKET="aether-releases"
```
Comment (obdulia-losantos) [inline scripts/deploy.sh]: this one too

Comment (MrPink) [reply scripts/deploy.sh]: what would I do without you 💯

Comment (obdulia-losantos) [reply scripts/deploy.sh]: I meant, `export RELEASE_BUCKET="aether-releases"`

File: scripts/deploy.sh [hunk 5]
```diff
@@ -55,14 +62,22 @@ elif [[ ${TRAVIS_BRANCH} =~ ^release\-[0-9]+\.[0-9]+$ ]]; then
     # deploy release candidates in ???
     GCS_PROJECT="alpha"
     GCR_PROJECT="development-223016"
+    export RELEASE_BUCKET="aether-releases-dev"
```
Comment (obdulia-losantos) [inline scripts/deploy.sh:65]: :+1:

File: scripts/deploy.sh (hunk 4 above)
Comment (obdulia-losantos) [inline scripts/deploy.sh]:
```suggestion
    export RELEASE_BUCKET="aether-releases"
```

**Review by obdulia-losantos — APPROVED** [review]

**Pull request merged**